topic-operator/src/test/scripts/reassign.sh

```diff
@@ -1,62 +0,0 @@
-#!/usr/bin/env bash
-
-# --fail MESSAGE
-# --verify-success PARTITION
-# --verify-fail PARTITION
-# --verify-in-progress PARTITION
-# --generate-current JSON
-# --generate-propose JSON
-# --execute-running
-# --execute-started
-# --execute-fail STR
-
-echo "Random logging-like rubbish"
```

Comment:
The purpose of the tests which use `reassign.sh` is to check we can execute a shell script and parse its output in a bunch of different circumstances. I can see that you can make that cross-platform by instead executing a Java program, but that changes what the test is testing. Really it does need to execute a shell script in particular (because as it stands that's what the TO will need to do). So I think the correct thing to do here, is to just `assumeLinux()` on all the tests which were using `reassign.sh`.

Reply:
Done